stdlib/in_channel.mli

```diff
@@ -0,0 +1,35 @@
+type t = in_channel
+
+val stdin : t
+
+val create : string -> t
```

Comment:
This opens a file, I want to see that in the code, I think these names should be `open'`, `open_bin` and `open_gen`.

Reply:
Right, fixed.